x-pack/legacy/plugins/lens/public/help_menu_util.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { HelpMenu } from './help_menu';
+
+export function addHelpMenuToAppChrome(chrome) {
```

Comment:
Why remove the file when it just needs to be plugged into the header's help menu? As stated in the summary of the PR, I just need someone to do that part. It can be pushed directly to this branch.